docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+ 
+5.	Make a new directory where your project will be generated:
+
+.. code-block:: console
+
+ mkdir c:\MyYo
+
+6.	On the command line, make the new directory the current directory.
+
+.. code-block:: console
+ 
+ cd c:\MyYo
+
+7.	Run the ``yo`` command and pass the name of the generator.
+
+.. code-block:: console 
+
+ yo aspnet
+
+8.	The aspnet generator displays a menu. Leave the default **Empty Application** type selected and press ``Enter``.
+
+9.	The generator prompts for the name of the application. Leave the default and press ``Enter``.
+
+.. image:: yeoman/_static/yeoman-yo-aspnet.png
+
+10.	Open Visual Studio 2015. From the File menu select :menuselection:`Open --> Project/Solution`.
+
+11.	In the Open Project dialog, navigate to the ``project.json`` file, select it and click the **Open** button. In the Solution Explorer, the project should look something like the screenshot below.
+
+ .. image:: yeoman/_static/yeoman-empty-application.png
+ 
+12.	This example doesn't use the cross-platform Kestrel server, so open ``project.json`` for editing and remove both references to **kestrel** from the **dependencies** and **commands** sections. Save the ``project.json`` file.
+
+13.	Press **F5** to run the project. The default Welcome screen should display.
+ 
+ .. image:: yeoman/_static/yeoman-welcome.png 
+ 
+Integrating Generated Projects with Visual Studio
+-------------------------------------------------
+
+Generator authors may not have Visual Studio in mind when they create a new Yeoman generator. Each generator implements its own distinct, opinionated, grouping of technology. As long as the generator creates a working set of files for the purpose, the generator is successful. For example, the `Yeoman team <http://yeoman.io/>`_ has a webapp generator that scaffolds a front-end web application. The generator includes a build process with Grunt or Gulp and Bower and a number of other features. The generated files have no concept of "project". The generated files (see screenshot below) don't contain a `project.json` and you won't find project files that are specific to Visual Studio.
+
+ .. image:: yeoman/_static/yeoman-generated-files.png  
+  
+To work on Yeoman generated files in Visual Studio, you have options:
+ 
+- Leave the file structure exactly as generated and use the Visual Studio :menuselection:`File --> Open --> Web Site` option to populate the solution as shown in the screenshot below. The web site option is the safest route for getting the project to work in Visual Studio initially. You can test the site from the command line before adding it to Visual Studio. If the site runs separately, it will run as a web site in Visual Studio. The Grunt and Gulp client-build tools will still work in Visual Studio, but, at the time of this writing, the **Dependencies** folder for NPM and Bower will not appear automatically. 
```

Comment:
Let's remove this content on using the old Web Site support and instead  focus on how to do this with an ASP.NET 5 project.